server/lib/settings/index.ts

```diff
@@ -379,6 +390,10 @@ class Settings {
         apiKey: '',
       },
       tautulli: {},
+      metadataSettings: {
+        tv: IndexerType.TMDB,
+        anime: IndexerType.TVDB,
```

Comment:
Why is this the default? Default should always be TMDB for both

Reply:
done